src/questions/css-questions.md

```diff
@@ -39,4 +39,6 @@ permalink: /questions/css-questions/index.html
 * Can you explain the difference between px, em and rem as they relate to font sizing?
 * Can you give an example of a pseudo class? Can you provide an example use case for a pseudo class? 
 * What is the difference between a block level element and an inline element. Can you provide examples of each type of element?
-* What is the difference between CSS Grid and Flexbox? When would you use one over the other?
\ No newline at end of file
+* What is the difference between CSS Grid and Flexbox? When would you use one over the other?
+* What are the Flexbox advantages compared to floats?
```

Comment:
You could reword this to be something like "Compare flexbox with legacy layout techniques."

Reply:
✅